config/routes.rb

```diff
@@ -4,7 +4,13 @@
  
```

Comment:
Empty whitespace, could be removed - as well as the comment near the end of the file

Reply:
Removed